apps/api/infra/api.ts

```diff
@@ -222,6 +222,11 @@ export const serviceSetup = (services: {
         '@rsk.is/prokura',
         '@rsk.is/prokura:admin',
       ]),
+      XROAD_RSK_PROCURING_PATH: {
+        dev: 'IS-DEV/GOV/10006/Skatturinn/relationships-v1',
+        staging: 'IS-TEST/GOV/10006/Skatturinn/relationships-v1',
```

Comment:
Still looks wrong. Staging and prod should have the national id of skatturinn in the url.

Reply:
Also why is this duplicated here? Isn't the api project using the shared xroad config below?

Reply:
Because of, but maybe I am wrong

https://app.datadoghq.eu/logs?query=service%3Aapi%20env%3Adev%20XROAD_RSK_PROCURING_PATH%20&cols=host%2Cservice&event=AgAAAYkgrKbqJizZ5gAAAAAAAAAYAAAAAEFZa2dyTFdtQUFDVjNfMUZuZ2FhSHdBQwAAACQAAAAAMDE4OTIwYWMtYjhiMy00NjYyLWJjZmItMjZhMTZlODgyNjM0&index=%2A&messageDisplay=inline&stream_sort=desc&viz=stream&from_ts=1688138964539&to_ts=1688398164539&live=true

![image](https://github.com/island-is/island.is/assets/112904566/781875ed-ce36-46c9-b10a-f591bb1d2ef4)

Reply:
My bad I did not see the `.xroad()` method. It has been imported into `infra/api.ts` and is now being shared